src/shared/Swagger/decorators/savedjobs/view-savedjobs.swagger.decorator.ts

```
@@ -0,0 +1,29 @@
+        ApiResponse({
+            status: HttpStatus.OK,
+            description: 'Sucesso na rota. Exemplo de sucesso: ',
+            type: ListResponseSwagger,
+        }),
+        ApiResponse({
+            status: HttpStatus.UNAUTHORIZED,
+            description: 'Não autorizado.',
+            type: UnauthorizedSwagger,
+        }),
+        ApiResponse({
+            status: HttpStatus.BAD_REQUEST,
```

ESLint

[error] Replace `············status:·HttpStatus.BAD_REQUEST,␍` with `······status:·HttpStatus.BAD_REQUEST,`

(prettier/prettier)